generator/dashboards/operational_monitoring_dashboard.py

```diff
@@ -129,7 +129,7 @@ def to_lookml(self, bq_client):
                         graph_index += 1
 
         if "alerts" in kwargs and kwargs["alerts"] is not None:
-            kwargs["alerts"]["row"] = (int(graph_index / 2) * 10,)
+            kwargs["alerts"]["row"] = int(graph_index / 2) * 10
```

Comment:
this value generates the following line, which should be of type `int` not `Tuple[int]`:

https://github.com/mozilla/looker-hub/blob/9296fcba2ea6efa810798f071e225f6f75c4b350/operational_monitoring/dashboards/bug_1759000_rollout_initial_rollout_of_tcp_release_91_99.dashboard.lookml#L425